migrations/V10__init_schema.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE header
+(
+    height    BIGINT  PRIMARY KEY,
+    hash      VARCHAR NOT NULL,
+    timestamp TIMESTAMP NOT NULL,
+
+    -- For convenience, we store the entire application-specific header type as JSON. Just like
+    -- `leaf.leaf` and `leaf.qc`, this allows us to easily reconstruct the entire header using
+    -- `serde_json`, and to run queries and create indexes on application-specific header fields
+    -- without having a specific column for those fields. In many cases, this will enable new
+    -- application-specific API endpoints to be implemented without altering the schema (beyond
+    -- possibly adding an index for performance reasons).
+    data JSONB NOT NULL
+);
+-- TODO this index should be unique, but HotShot does not currently give the application a way to
+-- put the block height or parent hash in the header, so headers are not guaranteed to be distinct.
+-- This will be fixed soon with the block header refactor.
+CREATE INDEX header_hash ON header (hash);
```

Comment:
Can we still make this unique and add some rule to reconcile duplicates (i.e. store the header with the most recent timestamp)?

Reply:
It's annoying because then you have to check if a header with the same hash already exists and use `UPDATE` instead of `INSERT` (`INSERT` will fail if there is a unique index with an already existing entry). I think it's simpler to leave it the way it is and just add the unique constraint once we have block height in the header